CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0-alpha.0
+
+* `FEAT`: rewrite project to new, [Preact](https://github.com/preactjs/preact)-based architecture
+
+### Breaking Changes
+
+* `PropertiesProvider#getTabs` is no longer used. Migrate to the new `PropertiesProvider#getGroups` API instead.

```

Comment:
Consider pointing to an example how to migrate (before -> after).

That could be a commit on the `MagicProperties` example, if need be.